squared_equation.rb

```diff
@@ -0,0 +1,20 @@
+def solve_equation(a, b, c)
+  d = b**2 - 4 * a * c
+  puts "D = #{d}"
+  return nil if d.negative?
+
+  results = []
+  results.push((-b + Math.sqrt(d)) / 2 * a)
```

Comment:
https://github.com/rubocop-hq/ruby-style-guide#spaces-and-operators
Don't use extra spaces between math operators.